test/e2e/pod_log_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPodLogs(t *testing.T) {
```

Comment:
to simplify the test, can't we just deploy Pod resources directly instead of using Deployment? alternatively we can use statefulset so that names of pods are deterministic.

Reply:
Yeah sounds good. Started using `StatefulSet`.